azure-pipelines.yml

```diff
@@ -0,0 +1,53 @@
+jobs:
+- job:
+
+  strategy:
+    matrix:
+      Mojave-Xcode-10.2.1:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '10.2.1'
+      Mojave-Xcode-11.0:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '11'
+
+  pool:
+    vmImage: $(IMAGE_POOL)
+
+  steps:
+
+  - script: sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
+    displayName: "Select Xcode $(XCODE_VERSION)"
+
+  - script: xcodebuild -version
+    displayName: "Version of xcodebuild"
```

Comment:
Let's combine these two steps and add the same debugging information:

```
      set -e
	      sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
	      echo "Xcode version: $(xcrun xcodebuild -version)"
	      echo "Xcode path: $(xcrun -f xcodebuild)"
	      echo "Xcode select path: $(xcode-select --print-path)"
```

Reply:
Done